config/env/development.example.env

```diff
@@ -39,3 +39,7 @@ RAISE_DELIVERY_ERRORS="false"
 PSQL_HOST=""
 PSQL_USERNAME=""
 PSQL_PASSWORD=""
+
+# REDIS
```

Comment:
You'll want to add this to `config/env/test.example.env` as well.

Reply:
Done.